product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.6-xml-message-enrichment/1.6.2-Modify-payload-by-adding-a-sibling-using-enrich-mediator/src/test/java/org.wso2.carbon.ei.scenario.test/AddSiblingTest.java

```diff
@@ -70,6 +70,48 @@ public void AddSiblingInline() throws IOException, XMLStreamException {
                 "urn:mediate", "AddSiblingInline");
     }
 
+    //This test is to verify if payload can be modified by adding sibling elements defined by xpath to the payload.
+    @Test(description = "Adding an inline content as a sibling to the message body")
```

Comment:
Let's add test case ID in the description

Reply:
Added